packages/esm-billing-app/src/billable-services/payment-history/payment-history-table.component.tsx

```diff
@@ -23,6 +23,7 @@ export const PaymentHistoryTable = ({
   const headers = [
     { header: t('billDate', 'Date'), key: 'dateCreated' },
     { header: t('patientName', 'Patient Name'), key: 'patientName' },
+    { header: t('id', 'Id'), key: 'identifier' },
```

Comment:
we should just be explicit here 
```suggestion
    { header: t('identifier', 'Identifier'), key: 'identifier' },
```